app/controllers/v0/claim_documents_controller.rb

```diff
@@ -47,7 +55,7 @@ def form_id
     end
 
     def unlock_file(file, file_password)
```

Comment:
is this something that should live in `lib/pdf_utilities` ?

Reply:
Agreed, definitely something that would make sense to move over.